.circleci/config.yml

```diff
@@ -47,3 +58,20 @@ jobs:
           name: run rubocop
           command: |
             bundle exec rubocop
+
+      - run:
+          name: Wait for DB
+          command: dockerize -wait tcp://localhost:5432 -timeout 1m
+
+      - run:
+          name: Database setup
+          command: bin/rails db:schema:load --trace
+
+      - run:
+          name: run espec
+          command: |
+            bundle exec rspec
+
+      # Save artifacts
+      - store_test_results:
```

Comment:
if using this directive, be sure to [follow the instructions on CircleCI](https://circleci.com/docs/2.0/collect-test-data/#enabling-formatters)